java/org/contikios/cooja/serialsocket/SerialSocketServer.java

```diff
@@ -400,7 +400,7 @@ public boolean startServer(int port) {
           try {
             serverSocket.close();
           } catch (IOException ex) {
-            logger.error(ex);
+            logger.error("Failed closing server socket: ", ex);
```

Comment:
Ending space is unnecessary as the stack trace is shown on its own rows. Same for the other log outputs with ending space.

Reply:
Fixed.